go.mod

```diff
@@ -45,6 +46,9 @@ require (
 )
 
 require (
+	dario.cat/mergo v1.0.1 // indirect
+	github.com/Masterminds/goutils v1.1.1 // indirect
+	github.com/Masterminds/semver/v3 v3.3.0 // indirect
```

Comment:
These dependencies are something we might want to prevent having.

Reply:
I think these came up while importing sprig library.

Reply:
Updated the dependency.